### DISCUSSION TITLE
kafka_consumergroup_current_offset_sum from kafka-exporter shows -1

### DISCUSSION THREAD

**Discussion by Casper-dss** [strimzi/strimzi-kafka-operator#10028] [General]
Hi 

The metric prometheus.metrics.kafka_consumergroup_current_offset_sum shows -1. I think it is calculating wrong, depending on what the purpose with this metric is.

We have a topic with 3 partitions and 1 message, and one consumer group, with on consumer. When I look at the metrics for each of the partitions it looks like this:
- par 0 = -1
- par 1 = -1
- par 2 = 1

When these are added, the result is -1. But shouldn't it be 1? I think the -1 values should be ignored in the sum.

**Comment by scholzj** [comment]
You should probably start by explaining what is the metric you are talking about, where do you get it etc.

**Reply by Casper-dss** [reply]
It is kafka_consumergroup_current_offset_sum from kafka exporter
